conf/local.conf.sample

```diff
@@ -230,6 +220,41 @@ BB_DISKMON_DIRS ??= "\
 #file://.* file:///some/local/dir/sstate/PATH"
 
 
+# Various packages dynamically add users and groups to the system at package
+# install time.  For programs that do not care what the uid/gid is of the
+# resulting users/groups, the order of the install will determine the final
+# uid/gid.  This can lead to non-deterministic uid/gid values from one build
+# to another.  Use the following settings to specify that all user/group adds
+# should be created based on a static passwd/group file.
+#
+# Note, if you enable or disable the useradd-staticids in a configured system,
+# the TMPDIR may contain incorrect uid/gid values.  Clearing the TMPDIR
+# will correct this condition.
+#
+# By default the system looks in the BBPATH for files/passwd and files/group
+# the default can be overriden by spefying USERADD_UID/GID_TABLES.
+#
+USERADDEXTENSION = "useradd-staticids"
+#USERADD_UID_TABLES = "files/passwd"
+#USERADD_GID_TABLES = "files/group"
+#
+# In order to prevent generating a system where a dynamicly assigned uid/gid
+# can exist, you should enable the following setting.  This will force the
+# system to error out if the user/group name is not defined in the
+# files/passwd or files/group (or specified replacements.)
+USERADD_ERROR_DYNAMIC = "1"
+
+# Enable internal source mirroring. Set SOURCE_MIRROR_URL externally.
+INHERIT += "own-mirrors"
+BB_GENERATE_MIRROR_TARBALLS = "1"
+
+# The network based PR service host and port
+# Uncomment the following lines to enable PRservice.
+# Set PRSERV_HOST to 'localhost:0' to automatically
+# start local PRService.
+# Set to other values to use remote PRService.
+PRSERV_HOST = "versionator.amer.corp.natinst.com:8585"
```

Comment:
Can you explain this config? Should it commented since the comment above says, "Uncomment...".

Reply:
I think the comment is a leftover from the example version of this file in OE upstream (ie. the line was originally commented-out). This variable defines the location of bitbake's Patch-Rev server, which bitbake will interrogate to figure out what the PR value should be for new recipe builds. Within the NI network, versionator will resolve to a sqlite database that we host on the gustavo hardware.

I don't think it hurts anything to have this variable defined; even outside of the NI network. If it does, we can add a toggle for it to the build environment init.

Reply:
Can you remove this line then?
"# Uncomment the following lines to enable PRservice."

Reply:
Done.